ui/components/multichain/address-copy-button/address-copy-button.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { ICON_NAMES, ButtonBase } from '../../component-library';
+import {
+  BackgroundColor,
+  TextVariant,
+  TextColor,
+  Size,
+  BorderRadius,
+  AlignItems,
+} from '../../../helpers/constants/design-system';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+import { shortenAddress } from '../../../helpers/utils/util';
+import Tooltip from '../../ui/tooltip/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const AddressCopyButton = ({
+  address,
+  shorten = false,
+  wrap = false,
+}) => {
+  const displayAddress = shorten ? shortenAddress(address) : address;
+  const [copied, handleCopy] = useCopyToClipboard();
+  const t = useI18nContext();
+
+  return (
+    <Tooltip position="bottom" title={copied ? t('copiedExclamation') : null}>
+      <ButtonBase
+        backgroundColor={BackgroundColor.primaryMuted}
+        onClick={() => handleCopy(address)}
+        paddingRight={3}
+        paddingLeft={3}
```

Comment:
Most of our buttons have 16px padding? Is this correct @SaraCheikh @Akatori-Design 
```suggestion
        paddingRight={4}
        paddingLeft={4}
```